lib/chr/generators/app_generator.rb

```diff
@@ -170,6 +171,14 @@ def init_git
     end
 
 
+    def create_deploy_scripts
+      say 'Creating deploy scripts'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.